openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryHeader/GlossaryHeader.component.tsx

```diff
@@ -143,9 +142,7 @@ const GlossaryHeader = ({
   const handleVersionClick = async () => {
     let path: string;
     if (isVersionView) {
-      path = isGlossary
-        ? getGlossaryPath(latestGlossaryData?.fullyQualifiedName)
-        : getGlossaryTermsPath(latestGlossaryData?.fullyQualifiedName ?? '');
+      path = getGlossaryPath(latestGlossaryData?.fullyQualifiedName);
```

Comment:
why did we remove glossary terms path here ?

Reply:
we aren't using those, We just determine glossary or glossaryTerm from the split function from FQN class